hw/top_darjeeling/top_darjeeling_ibex_pmp_reset_pkg.core

```diff
@@ -8,6 +8,9 @@ CAPI=2:
 # Tracked in https://github.com/olofk/fusesoc/issues/235
 name: "lowrisc:constants:top_darjeeling_ibex_pmp_reset_pkg"
 description: "Top-level specific PMP reset settings for Darjeeling"
+virtual:
+  - lowrisc:constants:ibex_pmp_reset_pkg
```

Comment:
Could you place a string implying it is virtual in this name?

Reply:
I think I'm going to get rid of this virtual dependency. This package is only consumed in the top level template, so there is no need for a virtual VLNV.